src/error.rs

```diff
@@ -162,3 +162,14 @@ impl fmt::Display for BorrowError {
 }
 
 impl_pyerr!(BorrowError);
+
+pub(crate) struct IgnoreError;
```

Comment:
Let's document the internal usage. Only a short note should be OK.

Reply:
Done.